From 7dfc1fc9450525cfcc3567f658db7e494b56aaf9 Mon Sep 17 00:00:00 2001
From: Jerry Jelinek <jerry.jelinek@joyent.com>
Date: Wed, 21 Feb 2018 20:01:52 +0000
Subject: [PATCH] OS-6651 root fs needs a bit more space for bhyve Reviewed by:
 Patrick Mooney <patrick.mooney@joyent.com> Reviewed by: Mike Gerdts
 <mike.gerdts@joyent.com> Approved by: Patrick Mooney
 <patrick.mooney@joyent.com>

---
 tools/build_live | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/build_live b/tools/build_live
index 77564714..569cf012 100755
--- a/tools/build_live
+++ b/tools/build_live
@@ -788,7 +788,7 @@ bi_setup_work_dir
 # Create and mount the "/" (root) ramdisk image:
 #
 bi_file_root="$bi_tmpdir/root.image"
-bi_root_size=272000
+bi_root_size=276000
 if strings "$bi_wsroot/proto/kernel/amd64/genunix" |
     grep "DEBUG enabled" >/dev/null; then
 	#
-- 
2.21.0

